src/test/regress/expected/citus_non_blocking_split_shards.out

```diff
@@ -15,6 +15,15 @@ Here is a high level overview of test plan:
  12. Show we allow Split with the shard transfer mode 'auto' if all colocated tables has replica identity.
 */
 CREATE SCHEMA "citus_split_test_schema";
+-- Since Deferred drop is enabled, set the cleanup interval to a large value
+-- to avoid flaky tests.
+ALTER SYSTEM SET citus.defer_shard_delete_interval TO 600000;
```

Comment:
better to set it to -1 to disable, we sometimes run our regression tests in valgrind, which slows them down to a crawl

Reply:
Updated.